crawl-ref/source/describe-spells.cc

```diff
@@ -499,6 +499,15 @@ static string _effect_string(spell_type spell, const monster_info *mon_owner)
         }
         if (you.immune_to_hex(spell))
             return "(immune)";
+
+        if (spell == SPELL_MINDBURST
+            || spell == SPELL_PAIN)
```

Comment:
When possible, it's better to avoid hardcoding specific constants. Instead, think about the pattern that you're trying to describe ("hex spells with associated damage zaps") and write your code to target that.